jiant/models.py

```diff
@@ -541,6 +543,9 @@ def build_task_specific_modules(task, model, d_sent, d_emb, vocab, embedder, arg
             params=task_params,
         )
         setattr(model, "%s_mdl" % task.name, module)
+    elif isinstance(task, SentenceOrderTask):
```

Comment:
Like I said before, this should go at the bottom of this if/else block, just for style/readability.